README.md

```diff
@@ -59,24 +59,10 @@ can be found in the [GitHub releases](https://github.com/Nheko-Reborn/nheko/rele
 #### Arch Linux
 
 ```bash
-pacaur -S nheko # nheko-git
```

Comment:
Can you maybe include a reference to the nheko-git package, which is still useful? :)

Reply:
Alright, done in eb1d874918575b5a6a042861163695d3559e5cf7